File: genFile.pl
#!/usr/bin/env perl

use strict;
use warnings;

sub genFile
{
	print "Starting Run:\n";

	for (my $count = 1; $count <=64; $count++)
	{
		my $filename = "tmp/$count";
		open(my $fh, '>', $filename) or die "Could not open file '$filename' $!";
		print $fh $count;
		close $fh;
	}
}

genFile();

File: genFile.py
#!/usr/bin/env python

def genFile():
    count = 0
    
    for count in range(64):
        count = count + 1
        tmptest = str(count)
        fileh = open("tmp/" + tmptest, "w")
        fileh.write(str(count))

if __name__ == '__main__':
    genFile()
